### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by Sencc** [YuYue525/MobileSAM-pytorch#4]
Hi,this works great!
I would like to ask if I need to train, how should I construct a custom data set? Does it need to be in the same format as SA-1B?

**Comment by YuYue525** [comment]
If you use this repo, it should be in the same format

**Comment by 781008xh** [comment]
Excuse me ,have you figured out how to train using your own dataset?

**Comment by 781008xh** [comment]
@Sencc Excuse me,have you figured out how to train using your own dataset?

**Comment by Sencc** [comment]
> @Sencc Excuse me,have you figured out how to train using your own dataset?

The training is very simple. You only need to prepare a data set, which contains the original image and the corresponding mask image.

**Comment by 781008xh** [comment]
@Sencc Is it possible to use a dataset that is not in the SA1B format？

**Comment by Sencc** [comment]
> @Sencc Is it possible to use a dataset that is not in the SA1B format？

Yes, you can build your own data set, but you may need to modify the data loading processing in the train code

**Comment by 781008xh** [comment]
@Sencc  okok，thank you!!!!!!
